src/lib.rs

```diff
@@ -247,14 +282,14 @@ pub trait PolynomialCommitment<F: Field>: Sized {
 
     /// Verifies that `values` are the evaluations at `point` of the polynomials
     /// committed inside `commitments`.
-    fn check<'a>(
+    fn check<'a, R: RngCore>(
         vk: &Self::VerifierKey,
         commitments: impl IntoIterator<Item = &'a LabeledCommitment<Self::Commitment>>,
         point: F,
         values: impl IntoIterator<Item = F>,
         proof: &Self::Proof,
         opening_challenge: F,
-        rng: Option<&mut dyn RngCore>,
+        rng: &mut R,
```

Comment:
This change also seems like a mixup during merge; can you change it to what's on master?

Reply:
I agree. This is a mixup that conflicts with the upstream change to make the rng optional.